tests/featureTesting/compare_csv.py

```diff
@@ -211,3 +191,45 @@ def get_layout(self, plot_title, x_title, y_title):
         )
 
         return layout
+
+    def get_correlation_and_wilcox_df(self, pandas_df):
+
+        unique_identifier_name_x_list = []
+        unique_identifier_name_y_list = []
+        rows_list = []
+        for index, row in pandas_df.iterrows():
+            x = []
+            y = []
+            unique_identifier_x_list = []
+            unique_identifier_y_list = []
+            for sample_name in self.sample_list:
+                sample_name_x = sample_name + "_x"
+                sample_name_y = sample_name + "_y"
+                x.append(row[sample_name_x])
+                y.append(row[sample_name_y])
+
+            unique_identifier_columns_x = ["compound_x", "isotopeLabel", "medRt_x"]
```

Comment:
This can be moved to constants, the entire list and accessed from there. Same for the `_y` list

Reply:
Done